cli/src/test/deploy.spec.ts

```diff
@@ -120,6 +120,15 @@ test('given --app-name but app not in list gives informative message', async (t)
   t.snapshot({ ...result, out: result.out.replace(t.context.projectDir, 'PROJECT_DIR') });
 });
 
+test('project associated deploys to new app with --new-app', async (t) => {
+  const newApp = createApp({ id: 'hybrid-koala' }); // hybrid-koala != fluffy-samaritan (default)
```

Comment:
What would happen if you passed `fluffy-samaritan`?  Should that also be a test??